windows/src/engine/kmcomapi/com/system/keymancontrol.pas

```diff
@@ -281,7 +281,7 @@ function TKeymanControl.IsConfigurationOpen: WordBool;
 begin
   with TKeymanMutex.Create('KeymanConfiguration') do
   try
-    Result := MutexOwned;
+    Result := TakeOwnership;
```

Comment:
I think this highlights a bug in existing usage -- `IsConfigurationOpen` should return true if the mutex is already owned. So ... this should be:

```suggestion
    Result := not TakeOwnership;
```

Am I right?